src/main/java/ru/skypro/homework/service/impl/UsersServiceImpl.java

```diff
@@ -46,17 +50,20 @@ public class UsersServiceImpl implements UsersService {
 
     private final UserRepository userRepository;
     private final UsersMapping usersMapping;
+    private final BCryptPasswordEncoder bCryptPasswordEncoder;
 
-    public UsersServiceImpl(UserRepository userRepository, UsersMapping userMapping) {
+    public UsersServiceImpl(UserRepository userRepository, UsersMapping usersMapping, BCryptPasswordEncoder bCryptPasswordEncoder) {
```

Comment:
```suggestion
    public UsersServiceImpl(UserRepository userRepository, UsersMapping usersMapping, PasswordEncoder bCryptPasswordEncoder) {
```
инжектить надо по интерфейсу, а не реализации